styles.css

```diff
@@ -93,7 +101,338 @@ ul.navbar-items {
+}
+
+.left-section {
+  flex: 1;
+}
+
+.left-section img {
+  float: left;
+  margin-right: 30px;
+  max-width: 100%;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  gap: 10px;
+  margin: 0 200px;
+}
+
+.grid-item {
+  position: relative;
+  background-color: #eaeaea;
+  height: 400px;
+}
+
+.first-desc h3 {
+  color: #091e42 !important;
+  font-size: 32px !important;
+  line-height: 44px !important;
+}
+
+.first-desc p {
+  font-size: 16px !important;
+  line-height: 25px;
+  text-align: left;
+}
+
+.first-desc ul li {
+  display: inline-block;
+  list-style: none;
+  border: 1px solid grey;
+  padding: 10px;
+  margin-right: 10px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.first-desc button {
+  background-color: #f55800;
+  border: none;
+  padding: 10px 10px;
+  margin-top: -40px !important;
+  font-family: 'Crete Round', sans-serif !important;
+  font-style: normal !important;
+  font-weight: 700px !important;
+  font-size: 16px !important;
+  line-height: 30px !important;
+  letter-spacing: 0.03em;
+  color: #eaeaea;
+}
+
+.grid-item:nth-child(1) {
+  background-image: url("./img/grid-1.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.grid-item:nth-child(2) {
+  background-image: url("./img/grid-2.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.grid-item:nth-child(3) {
+  background-image: url("./img/grid-3.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.grid-item:nth-child(4) {
+  background-image: url("./img/grid-4.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.grid-item:nth-child(5) {
+  background-image: url("./img/grid-5.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.grid-item:nth-child(6) {
+  background-image: url("./img/grid-6.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.item-child {
+  position: absolute;
+  bottom: 20px;
+  width: 100%;
+  text-align: left;
+}
+
+.items-details {
+  padding: 50px 50px 10px 10px;
+  text-align: left;
+}
+
+.item-child h3 {
+  color: #ffff;
+  text-align: left;
+  margin-left: 10px;
+  margin-bottom: 0;
+}
+
+.item-child p {
+  color: #ffff;
+  text-align: left;
+  margin-left: 10px;
+  margin-bottom: 0;
+}
+
+.item-child ul li {
+  background-color: rgba(255, 255, 255, 0.24);
+  display: inline-block;
+  list-style: none;
+  border: 1px solid grey;
+  padding: 5px;
+  margin-right: 10px;
+}
+
+.item-child button {
+  display: none;
+}
+
+.item-child ul {
+  padding-left: 0;
+  margin-left: 10px;
+}
+
+.last-btn {
+  color: #efe4e4;
+  background-color: #ff6b00;
+  padding-bottom: 40px;
+  text-align: center;
+  width: 100%;
+  height: 51px;
+  border: none;
+  font-family: 'Crete Round', sans-serif !important;
+  font-style: normal !important;
+  font-weight: 700px !important;
+  font-size: 16px !important;
+  line-height: 30px !important;
```

Comment:
- **[OPTIONAL]** KIndly avoid to use **!important** in your styles for the best practices of HTML & CSS.